### ISSUE TITLE
pdfLaTeX fails on pstool when hyperref is included

### ISSUE THREAD

**Issue by Tobster77** [MiKTeX/miktex#660]
Running ```pdfLaTeX``` from latest MikTeX (Windows) with ```psfragfig``` from the ```pstool``` package and ```hyperref``` leads to an error, while the same code works fine on TeXlive. Is this the rigt place to report the issue, or do I need to contact some package maintainer?

A minimum example, EPS is attached in ZIP archive:
```
\documentclass[12pt,a4paper]{article}
\usepackage[margin=2cm,a4paper]{geometry}
\usepackage{hyperref}

\title{psfrag in pdflatex }

\usepackage{graphicx}
\usepackage{pstool}
\usepackage{bookmark}


\begin{document}
Test
\begin{figure}[h!]
\centering
 \psfragfig[mode=errorstop,width=0.9\textwidth]{take_any_EPS}
 {
   \psfrag{t}[lc][lc]{psfrag}
 }
\caption{The EPS image with psfragfig}
\end{figure}

\end{document}
```

Now removing ```hyperref``` and ```bookmark``` resolves the issue and creates the correct PDF!

pdfTeX is version 3.14159265-2.6-1.40.21 on MiKTeX 20.11 Portable.
```hyperref``` is version 7.00e from May 17, 2020.
```pstool``` is version 1.5e from January 22, 2018.

[take_any_EPS.zip](https://github.com/MiKTeX/miktex/files/5533368/take_any_EPS.zip)

**Comment by edocevoli** [comment]
Cannot reproduce the issue without a complete test case. Please attach file `take_any_EPS`.

**Comment by Tobster77** [comment]
File is now attached to original post. I also set the ```psfrag``` labels and assigend them in the example code above.

**Comment by u-fischer** [comment]
It doesn't work for me in texlive (on windows) neither. I don't get the hyperref error, but ghostscript fails on the picture, due to an incompability between preview and hyperref that I reported here https://lists.gnu.org/archive/html/bug-auctex/2020-11/msg00000.html. 

There seem to be additionally a problem in the timing of pstool when moving the aux around.

I don't think that  any of these problems are miktex specific, but are problems of the involved packages triggered by the LaTeX changes in the shipout code and hook management.

I added also an answer to the tex.sx question: https://tex.stackexchange.com/a/570832/2388
